doc/reference/cmdline.rst

```diff
@@ -6,6 +6,21 @@
 Command-line interface: odoo-bin
 ================================
 
+.. _reference/cmdline/help:
+
+Help & version
+==============
+
+.. program:: odoo-bin
+
+.. option:: -h, --help
+
+    shows help text with all available options
+
+.. option:: --version
+
+   shows Odoo version e.g. "Odoo Server 13.0"
+
```

Comment:
Really not useful or necessary.

Reply:
I agree that it's not the most used feature, but I find it useful when checking server version especially with script (where the format of the response is also important).
It's your call whether to keep it or remove it, I will adapt the PR.
As this document is a reference I think it should show all options no matter how often are they used.